app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -55,7 +55,7 @@
 // $rem-base: 16px;
 
 // Allows the use of rem-calc() or lower-bound() in your settings
-@import 'foundation/functions';
+@import 'foundation/functions';  
```

Comment:
It doesn't look like any real changes happened in here. Can we `git checkout` this to minimize churn in the file?